eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/BitcoinCoreClient.scala

```diff
@@ -42,13 +46,47 @@ import scala.util.{Failure, Success, Try}
 
 /**
  * The Bitcoin Core client provides some high-level utility methods to interact with Bitcoin Core.
+ *
+ * @param rpcClient             bitcoin core JSON rpc client
+ * @param onchainKeyManager_opt optional onchain key manager. If provided and its wallet name matcher our rpcClient wallet's name, it will be used to sign transactions (it is assumed that bitcoin
```

Comment:
nit:

```suggestion
 * @param onchainKeyManager_opt optional onchain key manager. If provided and its wallet name matches our rpcClient wallet's name, it will be used to sign transactions (it is assumed that bitcoin
```

Reply:
done in https://github.com/ACINQ/eclair/pull/2613/commits/c5bb39f4043b8174ba3f372ae3ce30665523147d